engine/include/cubos/engine/input/plugin.hpp

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+#include <cubos/engine/cubos.hpp>
+
```

Comment:
I would include at least the `Input` class file here - when the user includes this header its basically guaranteed they also want to access the resource.